google-cloud-datastore/src/main/java/com/google/cloud/datastore/GqlQuery.java

```diff
@@ -494,26 +511,17 @@ public boolean equals(Object obj) {
   }
 
   com.google.datastore.v1.GqlQuery toPb() {
-    com.google.datastore.v1.GqlQuery.Builder queryPb =
-        com.google.datastore.v1.GqlQuery.newBuilder();
-    queryPb.setQueryString(queryString);
-    queryPb.setAllowLiterals(allowLiteral);
-    for (Map.Entry<String, Binding> entry : namedBindings.entrySet()) {
-      queryPb.putNamedBindings(entry.getKey(), entry.getValue().toPb());
-    }
-    for (Binding argument : positionalBindings) {
-      queryPb.addPositionalBindings(argument.toPb());
-    }
-    return queryPb.build();
+    GqlQueryProtoPreparer protoPreparer = new GqlQueryProtoPreparer();
+    return protoPreparer.prepare(this);
   }
 
   @Override
-  void populatePb(com.google.datastore.v1.RunQueryRequest.Builder requestPb) {
+  public void populatePb(com.google.datastore.v1.RunQueryRequest.Builder requestPb) {
     requestPb.setGqlQuery(toPb());
   }
 
   @Override
-  Query<V> nextQuery(com.google.datastore.v1.RunQueryResponse responsePb) {
+  public RecordQuery<V> nextQuery(com.google.datastore.v1.RunQueryResponse responsePb) {
```

Comment:
same here

Reply:
Fixed as part of https://github.com/googleapis/java-datastore/pull/823/commits/5f1e419a3e30aca687e49eadccd0a135a8f1b2e0.